### ISSUE TITLE
Backup/Restore keys instead of Export on Accounts Tab

### ISSUE THREAD

**Issue by tomusdrw** [paritytech/parity#4824]
Current "Export" button on Accoutns tab is a bit misleading. One may expect that it exports all accounts managed by Parity but it's not the case.
![screenshot from 2017-03-08 13-23-40](https://cloud.githubusercontent.com/assets/581548/23703916/a28317b0-0402-11e7-865d-1881a663b6ce.png)

The button actually exports only accounts data that seem kind of useless (NOTE: there is no "Import" button)
```
{
    "0x4d6Bb4ed029B33cF25D0810b029bd8B1A6bcAb7B": {
        "name": "Icarus Rain",
        "uuid": "434e0792-bda9-19e0-bd60-6d90e5235cac",
        "meta": {
            "timestamp": 1483691260203,
            "passwordHint": ""
        },
        "address": "0x4d6Bb4ed029B33cF25D0810b029bd8B1A6bcAb7B"
    }
}
```

I think it would rather make more sense to expose full key objects from the UI so that they can be imported on a different node/chain without touching any file system.

**Comment by thekevinb** [comment]
I second this.  
An fast usability fix could include 
"Export"=>"Metadata Export"
and a warning dialog about private keys not being included.

**Comment by ngotchac** [comment]
Closed via #4973
